web/style/common.scss

```diff
@@ -0,0 +1,284 @@
+      @content;
+    }
+  } @else if $point == desktop {
+    @media only screen and (min-width: 1024px) {
+      @content;
+    }
+  }
+}
+
+@mixin app-header-padding-changes {
+  @include break-point(desktop) {
+    padding: 0 $app-padding-horizontal;
+  }
+}
+
+.flex {
+  @include flex;
+}
+
+.flex-row {
+  @include flex-row;
+}
+
+// Common file 
+.flex-column {
+  @include flex-column;
+}
+
+.flex-row-reverse {
+  @include flex;
+  flex: 1;
+  flex-direction: row-reverse;
+}
+
+.flex-column-reverse {
+  @include flex;
+  flex: 1;
+  flex-direction: column-reverse;
+}
+
+*,
+*::after,
+*::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: inherit;
+  // color: $text-color-primary;
+}
+
+a {
+  color: $text-color-primary;
+  text-decoration: none;
+}
+
+html,
+body {
+  margin: 0 auto;
+  font-family: "Dosis";
+  font-weight: $app-font-weight;
+  box-sizing: border-box;
+}
+
+.btn {
+  background: $background-color-secondary;
+  border: 0;
+  padding: $app-btn-padding;
+  color: $text-color-secondary;
+  font-family: "Dosis";
+  text-align: center;
+}
+
+.nav {
+  display: flex;
+  height: 55px;
+  
+  @include break-point(tablet) {
+    height: $header-height;  
+  }
+  @include break-point(desktop) {
+    height: $header-height;  
+  }
+  box-shadow: 0 8px 6px -3px $grey-color-3;
+  @include app-header-padding-changes;
+
+  &__logo {
+    margin-left: 1vw;
+    width: 20vw;
+    padding: 9.5px 0;
+    @include break-point(tablet) {
+      margin-left: 0;
+      width: 23vw;
+    }
+
+    @include break-point(desktop) {
+      margin-left: 0;
+      width: 13vw;
+    }
+  }
+  &__bar {
+    @include flex-row;
+    list-style: none;
+    margin-left: 7vw;
+    padding: 9.5px 0;
+
+    &--link {
+      @include flex-column-reverse;
+      margin-left: 2vw;
+      display: none;
+      @include break-point(tablet) {
+        display: flex;
+      }
+      a {
+        color: #736a6d;
+      }
+    }
+  }
+
+  &__end {
+    @include flex;
+    flex-direction: column;
+    width: 29vw;
+    @include break-point(tablet) {
+      width: 12vw;
+    }
+
+    &-register-box {
+      @include flex;
+      // flex-direction: row-reverse;
+      height: 34%;
+      display: none;
+      @include break-point(tablet) {
+        display: flex;
+      }
+
+      &--link {
+        font-size: 0.7rem;
+        margin-left: 1.5vw;
+      }
+    }
+    &-cart {
+      @include flex-row;
+      justify-content: center;
+      align-items: center;
+      background-color: $grey-color-3;
+
+      &--logo {
+        height: 0.37 * $header-height;
+        fill: $background-color-secondary;
+      }
+    }
+  }
+}
+
+// toggle menu for mobile
+.menu-toggle {
+  display: block;
+  position: relative;
+  z-index: 1;
+  -webkit-user-select: none;
+  user-select: none;
+  align-self: center;
+  @include break-point(tablet) {
+    display: none;
+  }
+
+  input {
+    display: block;
+    width: 40px;
+    height: 32px;
+    position: absolute;

```

Comment:
Why all inputs in your application needs to be Absolute??

Reply:
No it's input of menu that is used for hamburger menu